app/Http/Requests/Api/Profile/UpdateRequest.php

```diff
@@ -27,8 +27,14 @@ public function authorize(): bool
     public function rules(): array
     {
         return [
-            'name'  => ['required', 'max:191', 'string'],
-            'email' => ['required', 'max:191', 'email', Rule::unique('users', 'email')->ignore($this->user())],
+            'name'  => ['required', 'string', 'max:191'],
+            'email' => [
+                'required',
+                'string',
+                'max:191',
+                'email',
+                Rule::unique('users', 'email')->ignore($this->user()),
```

Comment:
Isn't this going to return the validator message that makes it visible that this email already exists?